lib/src/main/java/com/auth0/jwt/JWTVerifier.java

```diff
@@ -58,12 +58,12 @@ static Verification init(Algorithm algorithm) throws IllegalArgumentException {
         /**
          * Require a specific Issuer ("iss") claim.
          *
-         * @param issuer the required Issuer value
+         * @param issuer the required Issuer value. If multiple values are given, the claim must at least match one of them
          * @return this same Verification instance.
          */
         @Override
-        public Verification withIssuer(String issuer) {
-            requireClaim(PublicClaims.ISSUER, issuer);
+        public Verification withIssuer(String... issuer) {
+            requireClaim(PublicClaims.ISSUER, Arrays.asList(issuer));
```

Comment:
You should not need to call `Arrays.asList` at all. If the varargs were used instead of a String, it would be an instance of Array which can already be considered a "list".

Similarly, if you could go ahead and fix the `withAudience` method which shouldn't be calling `Arrays.asList` as well. (same reason). Feel free to do it here or in a separate PR if desired.